etc/profile-a-l/linuxqq.profile

```diff
@@ -0,0 +1,45 @@
+# Firejail profile for linuxqq
+# Description: IM client based on Electron
+# This file is overwritten after every install/update
+# Persistent local customizations
+include linuxqq.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/QQ
+noblacklist ${HOME}/.mozilla
+
+mkdir ${HOME}/.config/QQ
+whitelist ${HOME}/.config/QQ
+whitelist ${HOME}/.mozilla/firefox/profiles.ini
+whitelist ${DESKTOP}
+
+include allow-bin-sh.inc
+include disable-shell.inc
+
+# Add the next line to your linuxqq.local if your kernel allows unprivileged userns clone.
+#include chromium-common-hardened.inc.profile
+
```

Comment:
```suggestion
```

Already in electron.profile.

Reply:
Correct. But if users do exactly what is suggested in the electron.profile comment and they add it to their `chromium-common.local` it will not be in the linuxqq sandbox profile chain right? That's why I added it here and changed to `linuxqq.local`. Would be different if it mentioned adding it in electron.local.